pydarn/plotting/maps.py

```diff
@@ -0,0 +1,481 @@
+                Default: True
+            colorbar_label: str
+                The label that appears next to the colour bar.
+                Requires colorbar to be true
+                Default: ''
+            title: str
+                Adds a title to the plot. If no title is specified,
+                one will be provided
+                Default: ''
+            len_factor: float
+                Normalisation factor for the vectors, to control size on plot
+                Larger number means smaller vectors on plot
+                Default: 150.0
+            kwargs: key=value
+                uses the parameters for plot_fov and projections.axis
+
+        """
+        # Find the record corresponding to the start time
+        if start_time is not None:
+            record = find_record(dmap_data, start_time, time_delta)
+        date = time2datetime(dmap_data[record])
+        if cmap is None:
+            cmap = {MapParams.FITTED_VELOCITY: 'plasma_r',
+                    MapParams.MODEL_VELOCITY: 'plasma_r',
+                    MapParams.RAW_VELOCITY: 'plasma_r',
+                    MapParams.POWER: 'plasma',
+                    MapParams.SPECTRAL_WIDTH: PyDARNColormaps.PYDARN_VIRIDIS}
+            cmap = plt.cm.get_cmap(cmap[parameter])
+        # Setting zmin and zmax
+        defaultzminmax = {MapParams.FITTED_VELOCITY: [0, 1000],
+                          MapParams.MODEL_VELOCITY: [0, 1000],
+                          MapParams.RAW_VELOCITY: [0, 1000],
+                          MapParams.POWER: [0, 250],
+                          MapParams.SPECTRAL_WIDTH: [0, 250]}
+        if zmin is None:
+            zmin = defaultzminmax[parameter][0]
+        if zmax is None:
+            zmax = defaultzminmax[parameter][1]
+
+        hemisphere = Hemisphere(dmap_data[record]['hemisphere'])
+        norm = colors.Normalize
+        norm = norm(zmin, zmax)
+
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            for stid in dmap_data[record]['stid']:
+                _, aacgm_lons, _, _, ax =\
+                        Fan.plot_fov(stid, date, ax=ax, **kwargs)
+        if parameter == MapParams.MODEL_VELOCITY:
+            data_lons = dmap_data[record]['model.mlon']
+            data_lats = dmap_data[record]['model.mlat']
+        else:
+            data_lons = dmap_data[record]['vector.mlon']
+            data_lats = dmap_data[record]['vector.mlat']
+
+            # Hold the beam positions
+            shifted_mlts = aacgm_lons[0, 0] - \
+                (aacgmv2.convert_mlt(aacgm_lons[0, 0], date) * 15)
+            shifted_lons = data_lons - shifted_mlts
+            mlons = np.radians(shifted_lons)
+            mlats = data_lats
+
+        # If the parameter is velocity then plot the LOS vectors
+        if parameter == MapParams.FITTED_VELOCITY:
+            v_mag, azm_v =\
+                    cls.calculated_fitted_velocities(mlats=mlats, mlons=mlons,
```

Comment:
```suggestion
                    cls.calculated_fitted_velocities(mlats=mlats, mlons=np.radians(data_lons),
```
Previously, the MLT was being passed into `calculate_fitted_velocities` instead of the actual magnetic longitude, which caused the velocity vectors to be wrong. This was first noticed by @Shirling-VT in the plotted vectors.